.env.local.template

```
@@ -1,6 +1,6 @@
-# Next secrets
-NEXTAUTH_URL=http://localhost:3000/
-NEXTAUTH_SECRET=SECRET
+# Next Auth secrets (v5 names; NEXTAUTH_SECRET / NEXTAUTH_URL also work as fallbacks)
+AUTH_URL=http://localhost:3000/
+AUTH_SECRET=SECRET
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTH_SECRET key should go before the AUTH_URL key

(UnorderedKey)